components/nds-card/src/Card.js

```diff
@@ -28,38 +28,51 @@ export const Card = props => {
 		headingText,
 		headingElementType,
 		link,
+		image,
+		children,
 		elementType: ContainerType = "article",
 		...rest
 	} = props;
 	const headerProps = { headingText, headingElementType, link };
 	return (
 		<ContainerType className="card" {...rest}>
-			<header className="card__header">
-				<CardHeader {...headerProps} />
-			</header>
-			{metadata && metadata.length && (
-				<dl className="card__metadata">
-					{metadata.map((item, idx) => {
-						if (!item.value) return null;
-						return (
-							<div key={`item${idx}`} className="card__metadatum">
-								{item.label && (
-									<dt className="visually-hidden">{item.label}</dt>
-								)}
-								<dd>{item.value}</dd>
-							</div>
-						);
-					})}
-				</dl>
+			{image && image.src && (
+				<img className="card__image" src={image.src} alt={image.alt} />
 			)}
+			<div className="card__text">
+				<header className="card__header">
+					<CardHeader {...headerProps} />
+				</header>
+				{children && <div className="card__summary">{children}</div>}
```

Comment:
I think if you pass in a p tag as part of children then this will result in an extra margin from the nested p? So we either need add css to remove these margins or swap children to a summary prop that just takes a string/React Element (element so we can add extra styling). I'd be in favour of the second because I don't think cards should have a description that's multiple paragraphs.

Reply:
I was playing with that. Initially it was a p but I thought a situation may occur where the summary has formatted text (bold etc) in it, if not multiple paragraphs. So I was thinking that it would be better to not have margins on the __summary wrapper and let the children inherit the default styling, which should all space out appropriately - shouldn't be any extra margins.

Reply:
Ah fair enough well if it works then cool!